### PR TITLE
Readme for new repo structure

### PR DESCRIPTION
**Pull request by sushilraje**
# Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] All new and existing tests passed.
- [ ] The code follows the code style and conventions of this project.
- [ ] The change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

# Description of the change

...

# Motivation for the change

...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,13 +64,7 @@ Architecture Overview
...
 * [Remote Monitoring Web UI](https://github.com/Azure/pcs-remote-monitoring-webui)
 * [Command Line Interface (CLI)](https://github.com/Azure/pcs-cli)
-* [IoT Hub manager](https://github.com/Azure/iothub-manager-dotnet)
-* [User Management](https://github.com/Azure/pcs-auth-dotnet)
-* [Device Simulation](https://github.com/Azure/device-simulation-dotnet)
-* [Telemetry](https://github.com/Azure/device-telemetry-dotnet)
-* [ASA Manager](https://github.com/Azure/asa-manager-dotnet)
-* [Configuration](https://github.com/azure/pcs-config-dotnet)
-* [Storage Adapter](https://github.com/azure/pcs-storage-adapter-dotnet)
+* [Micro Services](https://github.com/Azure/remote-monitoring-services-dotnet)
```
Comment (molly-moen) [inline README.md]: microservices is one word

File: docs/LOCAL_DEPLOYMENT.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+
+Local Deployment
+================
+### Overview
+This document and other documents, referenced here, contain steps to git clone the latest repository, deploying the microservices locally and also a walk through the general repository and microservice structure.
+&nbsp; 
+### Steps to clone the restructured repository
+#### If you have an existing cloned repository
+1) Use recurssive clone to clone the repository and the sub-modules.
```
Comment (molly-moen) [inline docs/LOCAL_DEPLOYMENT.md]: typo: recursive

File: docs/LOCAL_DEPLOYMENT.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+remote: Counting objects: 774, done.
+.....
+Submodule 'cli' (https://github.com/Azure/pcs-cli.git) registered for path 'cli'
+Submodule 'services' (https://github.com/Azure/remote-monitoring-services-dotnet.git) registered for path 'cli'
+Submodule 'webui' (https://github.com/Azure/azure-iot-pcs-remote-monitoring-webui.git) registered for path 'webui'
+.....
+````
+2) It is safe to delete exisiting microservice folders (except for services folder) under the top-level of the repository. 
+&nbsp; 
+#### If you have donot have an existing cloned repository
```
Comment (molly-moen) [inline docs/LOCAL_DEPLOYMENT.md]: do not should be two words

File: docs/LOCAL_DEPLOYMENT.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Description: 
+1) Scripts  
+The scripts folder is organized as follows\
+i. **docker** sub folder for building docker containers of the current microservice.\
+ii. **root** folder contains scripts for building and running services natively.\
+&nbsp; 
+![script folder structure](https://user-images.githubusercontent.com/39531904/44290937-10df4e00-a230-11e8-9cd4-a9c0644e166b.PNG "Caption")\
+&nbsp; 
+The docker build scripts require environment variables to be set up before execution. The run scripts can run both natively built and dockerized microservice. The run script under docker folder can also be independently used to pull and run published docker images. One can modify the tag and the account to pull different version or privately built docker images.
```
Comment (molly-moen) [inline docs/LOCAL_DEPLOYMENT.md]: There is an extra space at the beginning of this line

File: docs/LOCAL_DEPLOYMENT.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+abc@pcs git clone --recurse-submodules  https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet.git
+Cloning into 'azure-iot-pcs-remote-monitoring-dotnet'...
+remote: Counting objects: 774, done.
+.....
+Submodule 'cli' (https://github.com/Azure/pcs-cli.git) registered for path 'cli'
+Submodule 'services' (https://github.com/Azure/remote-monitoring-services-dotnet.git) registered for path 'cli'
+Submodule 'webui' (https://github.com/Azure/azure-iot-pcs-remote-monitoring-webui.git) registered for path 'webui'
+.....
+````
+2) It is safe to delete exisiting microservice folders (except for services folder) under the top-level of the repository.
```
Comment (molly-moen) [inline docs/LOCAL_DEPLOYMENT.md]: it might be good to specify that webui and cli should not be deleted

File: docs/LOCAL_DEPLOYMENT.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+4) WebService.Test  
+It contains unit tests for the REST endpoints of the microservice. 
+&nbsp; 
+
+5) Service  
+It contains unit tests for the business logic and code interfacing various SDKs.
+&nbsp;  
+
+6) Other Projects
```
Comment (molly-moen) [inline docs/LOCAL_DEPLOYMENT.md]: there isn't a new line for the description of other projects

File: docs/LOCAL_DEPLOYMENT.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+The microservice might contain other projects like RecurringTaskAgent etc.
```
Comment (molly-moen) [inline docs/LOCAL_DEPLOYMENT.md]: nit: "such as" instead of "like"

File: docs/LOCAL_DEPLOYMENT.md (hunk 2 above)
Comment (sushilraje) [reply docs/LOCAL_DEPLOYMENT.md]: Thanks for correcting, completed most of them

File: docs/LOCAL_DEPLOYMENT.md (hunk 3 above)
Comment (sushilraje) [reply docs/LOCAL_DEPLOYMENT.md]: done

File: docs/LOCAL_DEPLOYMENT.md (hunk 5 above)
Comment (sushilraje) [reply docs/LOCAL_DEPLOYMENT.md]: done

File: docs/LOCAL_DEPLOYMENT.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+Submodule 'webui' (https://github.com/Azure/azure-iot-pcs-remote-monitoring-webui.git) registered for path 'webui'
+.....
+````
+2) It is safe to delete existing microservice folders (except for services folder) under the top-level of the repository. Please do not delete web-ui, cli and reverse-proxy folders.
+&nbsp; 
+#### If you have do not have an existing cloned repository
+1) Follow the step 1  of "existing cloned repo". You will not require to delete anything. Ensure you have services sub-module, which contains all the microservices.
+&nbsp; 
+### Steps to deploy the microservices locally
+Follow the documentation [here](https://github.com/Azure/remote-monitoring-services-dotnet).
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: This link should take to local deploy document.

File: docs/LOCAL_DEPLOYMENT.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+&nbsp; 
+### Structure of repository and microservices
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: The structure of the microservices should be in the services repo. We should only keep top-level info here.

Comment (sushilraje) [reply docs/LOCAL_DEPLOYMENT.md]: ok, agreed

File: docs/LOCAL_DEPLOYMENT.md (hunk 8 above)
Comment (sushilraje) [reply docs/LOCAL_DEPLOYMENT.md]: I will have to do that once the deployment file is merged. I do not have permanent link.

File: docs/LOCAL_DEPLOYMENT.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+![top-level-scripts](https://user-images.githubusercontent.com/39531904/44433416-f4f2e980-a55a-11e8-8e3b-fc784788da58.PNG)\
+The docker build scripts require environment variables to be set before execution. 
+&nbsp; 
+
+2) Cli  
+This submodule contains cli source code, which can be used to deploy azure resources or create basic or standard deployments of the product 
+&nbsp;  
+
+3) Services 
+This submodule is link to the repository containing all the microservices.
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: Put a link to submodule repo once the other PR is merged.

File: docs/LOCAL_DEPLOYMENT.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+The scripts folder is organized as follows\
+i. **docker** scripts to deploy "released" tagged containers using docker-compose.\
+ii. **local** scripts to deploy "testing" tagged containers using docker-compose.\
+iii. **iothub** scripts to deploy create azure iot hub.\
+&nbsp; 
+![top-level-scripts](https://user-images.githubusercontent.com/39531904/44433416-f4f2e980-a55a-11e8-8e3b-fc784788da58.PNG)\
+The docker build scripts require environment variables to be set before execution. 
+&nbsp; 
+
+2) Cli
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: nit: Have link which opens the sub-module repo.

File: docs/LOCAL_DEPLOYMENT.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+
+2) Cli  
+This submodule contains cli source code, which can be used to deploy azure resources or create basic or standard deployments of the product 
+&nbsp;  
+
+3) Services 
+This submodule is link to the repository containing all the microservices.  
+&nbsp;
+
+4) Webui
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: ditto: about making this a link.

File: docs/LOCAL_DEPLOYMENT.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+This submodule is a link to ui component of the project. 
+&nbsp; 
+
+5) Reverse-proxy
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: nit: link

**Pull request merged**